sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -842,7 +842,7 @@ abstract class UnixTime extends ToTimestamp {
   examples = """
     Examples:
       > SELECT _FUNC_(0, 'yyyy-MM-dd HH:mm:ss');
-       1970-01-01 00:00:00
+       1969-12-31 16:00:00
```

Comment:
Oh, surprising.

Reply:
The `yyyy-MM-dd HH:mm:ss` pattern does not contain the time zone sub-pattern. If you point out it, you will something like:
```
spark-sql> SELECT from_unixtime(0, 'yyyy-MM-dd HH:mm:ssXXX');
1970-01-01 03:00:00+03:00
```

Reply:
And you can change your current time zone to `UTC` to see `1970-01-01 00:00:00`:
```sql
spark-sql> set spark.sql.session.timeZone=UTC;
spark.sql.session.timeZone	UTC
spark-sql> SELECT from_unixtime(0, 'yyyy-MM-dd HH:mm:ssXXX');
1970-01-01 00:00:00Z
```

Reply:
Ya. The timezone issue will make a failure on different timezone machines.

Reply:
but the time zone is forcibly set to `"America/Los_Angeles"` in tests:
https://github.com/apache/spark/blob/97dc4c0bfc3a15d364a376c6f87cb921d8d6980d/sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala#L36
https://github.com/apache/spark/blob/a1213d5f963f6e8815bbfaff308b0a24112fff54/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L1384